vision/object_detection_segmentation/retinanet/README.md

```diff
@@ -0,0 +1,98 @@
+# RetinanNet
+
+## Description
+[RetinaNet](https://github.com/NVIDIA/retinanet-examples) is a single-stage object detection model.
+
+## Model
+
+|Model        |Download  | Download (with sample test data)|ONNX version|Opset version|Accuracy |
+|-------------|:--------------|:--------------|:--------------|:--------------|:--------------|
+|RetinaNet (ResNet101 backbone)|    [228.4 MB](model/retinanet.onnx)   | [143.7 MB](model/retinanet.tar.gz)    |  1.6.0  |9| mAP 0.376      |
```

Comment:
can you include the opset in the names? i.e. model/retinanet-9.onnx, model/retinanet-9.tar.gz